README.md

```diff
@@ -6,10 +6,7 @@ This plugin provides client-side integration for the [CodePush service](http://c
 * [Supported React Native Platforms](#supported-react-native-platforms)
 * [Getting Started](#getting-started)
 * [iOS Setup](#ios-setup)
-    * [Plugin Installation](#plugin-installation-ios)
```

Comment:
Do we need to keep the old docs around for a little bit? Or are we expecting that anyone reading the getting started docs will be using the latest plugin version?